doc/governance.rst

```diff
@@ -20,95 +20,107 @@ the project community.
 Roles And Responsibilities
 ==========================
 
+We distinguish between contributors, core contributors, and the technical
+committee. A key distinction between them is their voting rights: contributors
+have no voting rights, whereas the other two groups all have voting rights,
+as well as permissions to the tools relevant to their roles.
+
 Contributors
 ------------
 
 Contributors are community members who contribute in concrete ways to the
 project. Anyone can become a contributor, and contributions can take many forms
 – not only code – as detailed in the :ref:`contributors guide <contributing>`.
+There is no process to become a contributor: once somebody contributes to the
+project in any way, they are a contributor.
+
+Core Contributors
+-----------------
+
+All core contributor members have the same voting rights and right to propose
+new members to any of the roles listed below. In order to join one of the roles
+bellow. Their membership is represented as being an organization member on the
```

Comment:
`... bellow.` - this sentence ended early/missing words? Also just one ell (l) in below.

Reply:
good catch.